specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/LinkedService.json

```diff
@@ -1433,7 +1433,51 @@
         "url"
       ]
     },
-    "AmazonS3LinkedService": {
+    "SapEccLinkedService": {
+      "x-ms-discriminator-value": "SapEcc",
+      "description": "Linked service for SAP ERP Central Component(SAP ECC).",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/LinkedService"
+        }
+      ],
+      "properties": {
+        "typeProperties": {
+          "description": "SAP ECC linked service properties.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SapEccLinkedServiceTypeProperties"
+        }
+      },
+      "required": [
+        "typeProperties"
+      ]
+    },
+    "SapEccLinkedServiceTypeProperties": {
+      "description": "SAP ECC linked service properties.",
+      "properties": {
+        "url": {
+          "type": "object",
```

Comment:
Same here and elsewhere.

Reply:
Similar here as explained above. Thanks.